build/config/BUILDCONFIG.gn

```diff
@@ -354,6 +360,8 @@ default_compiler_configs = [
   "//build/config/sanitizers:default_sanitizer_flags",
 ]
 
+default_compiler_configs += [ "//chrobalt/build/configs:chrobalt_config" ]
```

Comment:
Should this be inside of `if(chrobalt)` block ?

Reply:
added